### ISSUE TITLE
What is Open Education A11y: Headings

### ISSUE THREAD

**Issue by josieg** [BCcampus/open-sage#154]
**Describe the bug**
"What is open education?" appears twice as a h1 and a h2. It should only appear once, and all headings that appear after should be bumped up a level.

**To Reproduce**
Run WAVE on this page: https://helga.bccampus.ca/wp/opentextbooks/what-is-open-education/

**Expected behavior**
Headings should be unique:

- h1 What is Open Education
- h2 What are open textbooks
- h2 What are open educational resources

**Screenshots**
![Screenshot_2019-03-25 What is Open Education – open UAT site](https://user-images.githubusercontent.com/23706762/54932258-b9951080-4ed7-11e9-9874-b2d5ff1915bb.png)

**Comment by josieg** [comment]
This happens on a number of pages:

- https://helga.bccampus.ca/wp/opentextbooks/what-is-open-education/what-are-creative-commons-and-open-licences/
- https://helga.bccampus.ca/wp/opentextbooks/what-is-open-education/what-are-open-educational-resources/
- https://helga.bccampus.ca/wp/opentextbooks/what-is-open-education/what-are-open-textbooks/
- https://helga.bccampus.ca/wp/opentextbooks/what-is-open-education/what-is-open-pedagogy/
- https://helga.bccampus.ca/wp/opentextbooks/2-0-find-open-textbooks/

**Comment by josieg** [comment]
Closing this for now. As I figure out how the Customizer works I think there is a different fix for this.
